change/@azure-msal-node-a018e8b0-79a2-4e4b-b407-4488edfb0010.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Test changes",
```

Can you please provide a little more detail here? For example:

```suggestion
  "comment": "Tests for acquireTokenSilent #4672",
```

---

Sounds good, will keep it in mind!